src/vss_tools/vspec/vssexporters/vss2samm/helpers/sammConcepts.py

```diff
@@ -0,0 +1,108 @@
+# terms of the Mozilla Public License 2.0 which is available at
+# https://www.mozilla.org/en-US/MPL/2.0/
+#
+# SPDX-License-Identifier: MPL-2.0
+
+
+from enum import Enum
+from rdflib import URIRef
+
+from .. config import config as cnfg
+from . namespaces import samm_base_namespace, samm_output_namespace
+
+
+class VSSConcepts (Enum):
+    EMPTY               = ""                            # noqa: E221
+    BELONGS_TO          = "belongsToVehicleComponent"   # noqa: E221
+    HOLDS_VALUE         = "holdsState"                  # noqa: E221
+    HAS_SIGNAL          = "hasDynamicVehicleProperty"   # noqa: E221
+    HAS_ATTRIBUTE       = "hasStaticVehicleProperty"    # noqa: E221
+    PART_OF_VEHICLE     = "partOfVehicle"               # noqa: E221
+    PART_OF_VEH_COMP    = "partOf"                      # noqa: E221
+    HAS_COMP_INST       = "hasInstance"                 # noqa: E221
+    VEHICLE             = "Vehicle"                     # noqa: E221
+    VEHICLE_SIGNAL      = "ObservableVehicleProperty"   # noqa: E221
+    VEHICLE_ACT         = "ActuatableVehicleProperty"   # noqa: E221
+    VEHICLE_COMP        = "VehicleComponent"            # noqa: E221
+    VEHICLE_PROP        = "DynamicVehicleProperty"      # noqa: E221
+    VEHICLE_STAT        = "StaticVehicleProperty"       # noqa: E221
+
+    def __init__(self, vss_name):
+        self.ns = samm_output_namespace
+        self.vsso_name = vss_name
+
+    @property
+    def uri(self):
+        return URIRef(self.uri_string)
+
+    @property
+    def uri_string(self):
+        return f'{self.ns}{self.value}'
+
+
+class SammConcepts (Enum):
+    ASPECT                      = "Aspect"          # noqa: E221
+    PROPERTY                    = "Property"        # noqa: E221
+    ENTITY                      = "Entity"          # noqa: E221
+    CHARACTERISTIC              = "Characteristic"  # noqa: E221
+    NAME                        = "name"            # noqa: E221
+    CHARACTERISTIC_RELATION     = "characteristic"  # noqa: E221
+    PREFERRED_NAME              = "preferredName"   # noqa: E221
+    PAYLOAD_NAME                = "payloadName"     # noqa: E221
+    DESCRIPTION                 = "description"     # noqa: E221
+    PROPERTIES                  = "properties"      # noqa: E221
+    OPERATIONS                  = "operations"      # noqa: E221
+    EVENTS                      = "events"          # noqa: E221
+    DATA_TYPE                   = "dataType"        # noqa: E221
+    OPTIONAL                    = "optional"        # noqa: E221
+    EXAMPLE_VALUE               = "exampleValue"    # noqa: E221
+
+    def __init__(self, vss_name):
+        self.ns = f"{samm_base_namespace}:meta-model:{cnfg.SAMM_VERSION}#"
+        self.vsso_name = vss_name
+
+    @property
+    def uri(self):
+        return URIRef(self.uri_string)
+
+    @property
+    def uri_string(self):
+        return f'{self.ns}{self.value}'
+
+
+class SammCConcepts (Enum):
```

Comment:
Looks pretty similar if not the same  as `SammConcepts` Maybe define a common parent class?

Reply:
The SammConcepts and SammCConcepts use different uri's (namespaces):

SammConcepts has: self.ns = f"{samm_base_namespace}**:meta-model:**{cnfg.SAMM_VERSION}#"

where

SammCConcepts has: self.ns = f"{samm_base_namespace}**:characteristic:**{cnfg.SAMM_VERSION}#"

I will see how can I combine these and override just their initialization part with these namespaces.

Thanks for the catch :)